docs/manual/cli.md

```diff
@@ -55,7 +86,7 @@ preform authentication saving the need of explicitly using the `AUTH` command:
     $ redis-cli -a myUnguessablePazzzzzword123 PING
     PONG
 
-For safety it is strongly advised to provide the password to `redis-cli` automatically via the
+> **NOTE:** For security reasons, provide the password to `redis-cli` automatically via the
```

Comment:
Why the leading `>`?

We should standardize how we do notes (shortcodes is probably our way to go)

Reply:
remnant of an old practice of mine. it adds a line to the side. can remove. no problem.